1.0/docs/devguide/properties.md

```diff
@@ -325,7 +325,7 @@ Example:
     });
 
 **Warning:** A single property observer shouldn't rely on any other properties,
-sub-properties, or paths because the observer can be called while these 
```

Comment:
Is my text editor adding a special character or something to the end of the lines?